syft/pkg/cataloger/java/zip_wrapped_archive_parser.go

```diff
@@ -0,0 +1,43 @@
+package java
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/anchore/syft/internal/file"
+
+	"github.com/anchore/syft/syft/artifact"
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/pkg/cataloger/common"
+)
+
+// integrity check
+var _ common.ParserFn = parseZipWrappedJavaArchive
```

Comment:
what is this for?

Reply:
it isn't providing any functionality, but is instead a hint to developers working in this file that this function is meant to keep the same contract as a `common.ParserFn`.